action/lib/Database.js

```diff
@@ -60,4 +63,40 @@ module.exports = function(dbURL, dbName) {
                 });
             })
     };
+
+    this.getTriggerAssignment = function(workers) {
+        // a map between available workers and their number of assigned triggers
```

Comment:
no need to do fetch the view and do all this work if available worker list is not > 1